velociraptor.ts

```diff
@@ -32,13 +32,13 @@ export default <ScriptsConfiguration> {
     self_update: {
       cmd: "scripts/self_update.ts",
       desc:
-        "[INTERNAL] Self update your deno-scripts local repository with deno, instead of the bin",
+        "[INTERNAL] Self update your Denodarse local repository with deno, instead of the bin",
       ...common,
       allow: ["env", "read", "write", "run"],
     },
     create_self_update_bin: {
       cmd:
-        "deno install --force --allow-run --name deno_scripts_self_update scripts/self_update.ts",
+        "deno install --force --name deno_scripts_self_update scripts/self_update.ts",
```

Comment:
don't we need to use the --allow-run here to run a subprocess?

Reply:
Yes, but it is already defined in the `allow` property of the script. Velociraptor was throwing an error because of the duplicity of `allow-run` flag.